content/tutorial/02-advanced-orders/01-eth-flow/01-create-eth-flow/app-b/src/lib/run.ts

```diff
@@ -0,0 +1,108 @@
+import type { Web3Provider } from '@ethersproject/providers'
+import { Contract } from '@ethersproject/contracts'
+
+import { OrderBookApi, UnsignedOrder, OrderKind, SigningScheme, OrderCreation } from '@cowprotocol/cow-sdk'
+import { MetadataApi, latest } from '@cowprotocol/app-data'
+
+import { getSafeSdkAndKit } from './getSafeSdkAndKit'
+import { SETTLEMENT_CONTRACT_ABI, SETTLEMENT_CONTRACT_ADDRESS } from './const'
+
+export async function run(provider: Web3Provider): Promise<unknown> {
+  const safeAddress = '<PUT_YOUR_SAFE_ADDRESS>'
+  const appCode = '<YOUR_APP_CODE>'
+  const environment = 'prod'
+
+  // Slippage percent, it's 0 to 100
+  const quote = { slippageBips: '50' }
```

Comment:
Actually, it's in basis points ([bips](https://www.investopedia.com/ask/answers/what-basis-point-bps/)) witch is percent/100, so from 0 to 10000.
`50` -> 0.5%

Reply:
Am removing in-line comments in general (as exhaustive comments in the "walkthrough") so this will be taken care of with the advanced orders.